Winget-AutoUpdate-Install.ps1

```diff
@@ -58,6 +64,7 @@ Run WAU on metered connection. Default No.
 param(
     [Parameter(Mandatory = $False)] [Alias('S')] [Switch] $Silent = $false,
     [Parameter(Mandatory = $False)] [Alias('Path')] [String] $WingetUpdatePath = "$env:ProgramData\Winget-AutoUpdate",
+    [Parameter(Mandatory = $False)] [Alias('List')] [String] $ListPath = $WingetUpdatePath,

```

Comment:
I think you can leave it blank by default. This way it will behave as currently.

Reply:
No, this is required, it's used to determine if to call the Function or not:
if ($WingetUpdatePath -ne $ListPath){
                $NoClean = Test-ListPath $ListPath $UseWhiteList
            }

Reply:
in that case, simply do `if ($WingetUpdatePath)` no?

Reply:
Great, thanks!